ui/src/app/base/selectors/pod/pod.ts

```diff
@@ -137,7 +137,7 @@ const getHost = createSelector(
   [machine.all, controller.all, (_: RootState, pod: Pod) => pod],
   (machines: Machine[], controllers: Controller[], pod: Pod) => {
     if (!(pod && pod.host)) {
-      return null;
+      return undefined;
```

Comment:
Why this change, I would have thought `null` (absence of value) made more sense?

Reply:
This was a "strict mode" change. The selector returns a `Host`, which can be either a `Machine` or a `Controller`, or it doesn't exist (i.e. `undefined`), but it can't be `null`. It's a bit pedantic but it makes sense, or at least more sense than `type Host =  Machine | Controller | null`.

Reply:
Changed it so that the selector returns `Host | null` instead